src/suiryoku/locomotion/process/locomotion.cpp

```diff
@@ -764,7 +764,7 @@ bool Locomotion::position_kick_custom_pan_tilt(const keisan::Angle<double> & dir
   printf("delta tilt pan %.1f\n", delta_tilt_pan);
 
   double x_speed = 0.0;
-  if (!tilt == keisan::clamp(tilt, min_tilt.degree(), max_tilt.degree())) {
+  if (tilt != keisan::clamp(tilt, min_tilt.degree(), max_tilt.degree())) {
```

Comment:
What is this intended to do? is it the same like this?
```suggestion
  if (tilt < min_tilt.degree() || tilt > max_tilt.degree()) {
```

Reply:
yes, basically same